docs/serving/istio-authorization.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Enabling requests to Knative services when additional authorization policies are enabled"
+weight: 25
+type: "docs"
+---
+
+Knative Serving system pods, such as the activator and autoscaler components, require access to your deployed Knative services.
```

Comment:
What is meant by "access" exactly?

Reply:
The access for health checking and metric collection. I added them as `such as ...`.